app/controllers/payments_controller.rb

```diff
@@ -18,7 +18,7 @@ def create
 
     if @payment.purchase && @payment.save
       update_purchased_ticket_purchases
-      redirect_to conference_conference_registration_path(@conference.short_title),
+      redirect_to conference_physical_ticket_index_path,
```

Comment:
So like `update_purchased_ticket_purchases` is still excuting in the background and the user is redirected to index page? I am not sure if redirect_to works synchronously.